src/Tools/dotnet-monitor/Program.cs

```diff
@@ -76,6 +77,22 @@ class Program
                 Argument = new Argument<bool>(name: "noAuth", getDefaultValue: () => false)
             };
 
+        private static Option ShowRedactedConfig() =>
```

Comment:
I was expecting this to be a new command rather than flags added to the existing collect command. Here, you have, for example:

`dotnet monitor collect <other args> --redacted-config`

whereas I was expecting something like:

`dotnet monitor printconfig <other args> --level redacted`

I personally don't see the configuration printing analogous to a dry-run notion because other tools that use dry-run actually execute whatever the command is without effecting external state whereas this is just printing the understood configuration.

@shirhatti, do you have an opinion on using the existing collect command vs adding a new command (or some other option)?

Reply:
Had offline discussion, we landed on something like `dotnet monitor config show --level [Redacted|Full]` where the level parameter is optional.